ci-runner-image/Dockerfile

```diff
@@ -34,7 +34,15 @@ ENV PATH=$PATH:/usr/local/go/bin
 RUN apt-get install -y gnupg software-properties-common
 RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -
 RUN add-apt-repository --yes --update "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
-    && apt install -y terraform
+    && apt install -y terraform \
```

Comment:
why the final \?

Reply:
You are right, It's a typo, it worked anyway that's why I didn't realize, I am removingit, thanks.